src/server/handler_network_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/docker/docker/pkg/namesgenerator"
```

Comment:
`github.com/moby/moby/pkg/namesgenerator` and don't forget add to vendor.json

Reply:
it already add.